### PR TITLE
New Swedish translation.

### PR DESCRIPTION
**Pull request by sebras**
Because Adrian Black featured OpenHantek6022 in his latest [two](https://www.youtube.com/watch?v=8ts5J09Y7Gc) [videos](https://www.youtube.com/watch?v=PyFeBAcg7ls) I learned of this open source effort.

I already have an oscilloscope, but I wanted to contribute to the effort so I have translated OpenHantek6022 to Swedish.

### CODE REVIEW

File: openhantek/translations/openhantek_sv.ts [hunk 1]
```diff
@@ -0,0 +1,1343 @@
...
+    </message>
+    <message>
+        <location filename="../src/configdialog/DsoConfigAnalysisPage.cpp" line="9"/>
+        <source>Triangular</source>
+        <translation>Triangulär</translation>
+    </message>
+    <message>
+        <location filename="../src/configdialog/DsoConfigAnalysisPage.cpp" line="9"/>
+        <source>Gauss</source>
+        <translation>Gaus</translation>
```
Comment (haaninjo) [inline openhantek/translations/openhantek_sv.ts:52]: Gauss

File: openhantek/translations/openhantek_sv.ts [hunk 2]
```diff
@@ -0,0 +1,1343 @@
...
+    </message>
+    <message>
+        <location filename="../src/configdialog/DsoConfigScopePage.cpp" line="11"/>
+        <source>Set slowest possible timebase</source>
+        <translation>Sätt långsammast möjliga tidbas</translation>
+    </message>
+    <message>
+        <location filename="../src/configdialog/DsoConfigScopePage.cpp" line="18"/>
+        <source>Minimal time between captured frames&lt;br/&gt;(Longer times reduce the CPU load)</source>
+        <translation>Minimal tid mellan infågade bilder&lt;br/&gt;(Längre tider reducerar CPU-belastningen)</translation>
```
Comment (haaninjo) [inline openhantek/translations/openhantek_sv.ts:294]: infångade

File: openhantek/translations/openhantek_sv.ts [hunk 3]
```diff
@@ -0,0 +1,1343 @@
...
+    </message>
+    <message>
+        <location filename="../src/configdialog/DsoConfigScopePage.cpp" line="52"/>
+        <source>Position</source>
+        <translation>Position</translation>
+    </message>
+    <message>
+        <location filename="../src/configdialog/DsoConfigScopePage.cpp" line="54"/>
+        <source>Left</source>
+        <translation>Väsnter</translation>
```
Comment (haaninjo) [inline openhantek/translations/openhantek_sv.ts:324]: Vänster

File: openhantek/translations/openhantek_sv.ts [hunk 4]
```diff
@@ -0,0 +1,1343 @@
...
+    </message>
+    <message>
+        <location filename="../src/configdialog/DsoConfigScopePage.cpp" line="62"/>
+        <source>Cursors</source>
+        <translation>Markörer</translation>
+    </message>
+    <message>
+        <location filename="../src/configdialog/DsoConfigScopePage.cpp" line="66"/>
+        <source>Export zoomed screen in double height</source>
+        <translation>Expotera zoomad skärm med dubbel höjd</translation>
```
Comment (haaninjo) [inline openhantek/translations/openhantek_sv.ts:339]: Exportera

File: openhantek/translations/openhantek_sv.ts [hunk 5]
```diff
@@ -0,0 +1,1343 @@
...
+    </message>
+    <message>
+        <location filename="../src/configdialog/DsoConfigScopePage.cpp" line="71"/>
+        <source>Export</source>
+        <translation>Exportera</translation>
+    </message>
+    <message>
+        <location filename="../src/configdialog/DsoConfigScopePage.cpp" line="75"/>
+        <source>Save settings on exit</source>
+        <translation>Spara inställningar vad avslut</translation>
```
Comment (haaninjo) [inline openhantek/translations/openhantek_sv.ts:349]: vid avslut

File: openhantek/translations/openhantek_sv.ts [hunk 6]
```diff
@@ -0,0 +1,1343 @@
...
+    </message>
+    <message>
+        <location filename="../src/configdialog/DsoConfigScopePage.cpp" line="77"/>
+        <source>Apply default settings after next restart</source>
+        <translation>Tillämpla standardinställningar vid nästa omstart</translation>
```
Comment (haaninjo) [inline openhantek/translations/openhantek_sv.ts:354]: Tillämpa

File: openhantek/translations/openhantek_sv.ts [hunk 7]
```diff
@@ -0,0 +1,1343 @@
...
+    </message>
+    <message>
+        <location filename="../src/mainwindow.ui" line="122"/>
+        <source>Ctrl+Q</source>
+        <translation>Ctrl+Q</translation>
+    </message>
+    <message>
+        <location filename="../src/mainwindow.ui" line="130"/>
+        <source>S&amp;ampling</source>
+        <translation>s&amp;ampling</translation>
```
Comment (haaninjo) [inline openhantek/translations/openhantek_sv.ts:687]: Kan börja med stort S.

File: openhantek/translations/openhantek_sv.ts [hunk 8]
```diff
@@ -0,0 +1,1343 @@
...
+    </message>
+    <message>
+        <location filename="../src/mainwindow.cpp" line="124"/>
+        <source>Print screen ..</source>
+        <translation>Skriv ut skärm ..</translation>
+    </message>
+    <message>
+        <location filename="../src/mainwindow.cpp" line="186"/>
+        <source>Invalid command</source>
+        <translation>Ogiltig kommando</translation>
```
Comment (haaninjo) [inline openhantek/translations/openhantek_sv.ts:797]: Ogiltigt

File: openhantek/translations/openhantek_sv.ts [hunk 9]
```diff
@@ -0,0 +1,1343 @@
...
+    </message>
+    <message>
+        <location filename="../src/selectdevice/selectsupporteddevice.cpp" line="70"/>
+        <source>&lt;hr/&gt;&lt;p&gt;Even without a device you can explore the program&apos;s function. Just press the &lt;b&gt;Demo Mode&lt;/b&gt; button below.&lt;/p&gt;</source>
+        <translation>&lt;hr/&gt;&lt;p&gt;Även utan en enhet kan du utforska programmets funktioner. Tryck bara på knappen &lt;b&gt;Demoäge&lt;/b&gt; nedan.&lt;/p&gt;</translation>
+    </message>
+    <message>
+        <location filename="../src/selectdevice/selectsupporteddevice.cpp" line="85"/>
+        <source>&lt;p&gt;Upload in progress ...&lt;/p&gt;&lt;p&gt;&lt;b&gt;If the upload takes more than 30 s, please close this window &lt;br/&gt;and restart the program!&lt;/b&gt;&lt;/p&gt;</source>
+        <translation>&lt;p&gt;Uppsändning pågår ...&lt;/p&gt;&lt;p&gt;&lt;b&gt;Om sändningn tar mer än 30s, stäng detta fönster &lt;br/&gt;och starta om programmet!&lt;/b&gt;&lt;/p&gt;</translation>
```
Comment (haaninjo) [inline openhantek/translations/openhantek_sv.ts:1246]: Om sändning tar

**Review by haaninjo — COMMENTED** [review]
Nice as always, only some typos to fix :-)

**Comment by Ho-Ro** [comment]
Thank you for your enhancement - I merged it via copy/paste due to merge conflicts caused by previous local changes.

**Pull request closed without merging**